### PR TITLE
Add section "reviews" for mobile

### PR DESCRIPTION
**Pull request by taras-kolomiiets**
Add mvp section "reviews" for mobile )

### CODE REVIEW

File: src/partials/customer-reviews.html [hunk 1]
```diff
@@ -1 +1,45 @@
-<section id="customer-reviews" class="customer-reviews">Customer-reviews</section>
+<section id="customer-reviews" class="customer-reviews">
+  <div class="container">
+    <h2 class="visually-hidden">Customer reviews</h2>
+    <img
+      src="./images/reviews/curly-woman.jpg"
+      alt="Curly woman"
+      width="85px"
```
Comment (Mandzirokha) [inline src/partials/customer-reviews.html]: ширину картинки вказуємо без px

File: src/sass/layouts/_customer-reviews.scss [hunk 2]
```diff
@@ -1,3 +1,185 @@
 .customer-reviews {
-  color: yellow;
+  margin-top: 80px;
```
Comment (izachok) [inline src/sass/layouts/_customer-reviews.scss]: change to padding

File: src/sass/layouts/_customer-reviews.scss [hunk 3]
```diff
@@ -1,3 +1,185 @@
...
+}
+.customer-reviews__gradient-wrap {
+  position: relative;
+  width: 180px;
+  height: 248px;
+
+  margin-left: auto;
+  margin-right: auto;
+
+  background-color: var(--slider-backg-color);
```
Comment (izachok) [inline src/sass/layouts/_customer-reviews.scss]: background-color was inherited from parent. delete it

File: src/sass/layouts/_customer-reviews.scss [hunk 4]
```diff
@@ -1,3 +1,185 @@
...
+    height: 138px;
+
+    margin-bottom: 15px;
+    padding: 32px 52px;
+  }
+}
+.customer-reviews__gradient-wrap {
+  position: relative;
+  width: 180px;
+  height: 248px;
```
Comment (izachok) [inline src/sass/layouts/_customer-reviews.scss]: I think text should also be responsive in range 320-480. Only paddings should regulate it's position

File: src/partials/customer-reviews.html [hunk 5]
```diff
@@ -1 +1,45 @@
...
+    <img
+      src="./images/reviews/curly-woman.jpg"
+      alt="Curly woman"
+      width="85"
+      class="customer-reviews__image"
+    />
+    <div class="customer-reviews__text-wrap">
+      <div class="customer-reviews__gradient-wrap">
+        <svg class="customer-reviews__icon-brackets" width="54px" height="40px">
+          <use href="./images/icons.svg#icon-brackets"></use>
```
Comment (izachok) [inline src/partials/customer-reviews.html]: it should be pseudoelement by requirements

File: src/sass/layouts/_customer-reviews.scss [hunk 6]
```diff
@@ -1,3 +1,185 @@
...
+}
+.customer-reviews__text {
+  position: relative;
+  z-index: 2;
+
+  width: 157px;
+  margin-left: auto;
+  margin-right: auto;
+
+  font-family: 'DMSans-Medium';
```
Comment (izachok) [inline src/sass/layouts/_customer-reviews.scss]: is it possible change it to variable?

File: src/partials/customer-reviews.html [hunk 7]
```diff
@@ -1 +1,45 @@
...
+        </svg>
+        <p class="customer-reviews__text">
+          Nullam viverra consectetuer. Quisque cursus et, porttitor risus. Aliquam sem. In hendrerit
+          nulla quam nunc, accumsan congue. Lorem ipsum primis in nibh vel risus.
+        </p>
+      </div>
+    </div>
+    <div class="customer-reviews__dot-wrap">
+      <svg class="customer-reviews__icon-dot" width="4px" height="4px">
+        <use href="./images/icons.svg#icon-circle"></use>
```
Comment (izachok) [inline src/partials/customer-reviews.html]: it should be pseudoelement by requirements

File: src/sass/layouts/_customer-reviews.scss (hunk 2 above)
Comment (izachok) [inline src/sass/layouts/_customer-reviews.scss]: should you add bottom padding as well?

**Review by izachok — CHANGES_REQUESTED** [review]
Visually looks great

**Review by izachok — APPROVED** [review]

**Pull request merged**